climada/hazard/storm_europe.py

```diff
@@ -81,13 +82,18 @@ class StormEurope(Hazard):
     vars_opt = Hazard.vars_opt.union({'ssi_wisc', 'ssi', 'ssi_full_area'})
     """Name of the variables that aren't need to compute the impact."""
 
-    def __init__(self):
+    def __init__(self,
+                 units: Optional[str] = 'm/s',
+                 ssi: Optional[np.ndarray] = None,
+                 ssi_wisc: Optional[np.ndarray] = None,
+                 ssi_full_area: Optional[np.ndarray] = None
+                 ):
         """Calls the Hazard init dunder. Sets unit to 'm/s'."""
         Hazard.__init__(self, HAZ_TYPE)
```

Comment:
parameter description is missing in pydoc string

Reply:
I added a bit, but I need some help. I have no idea what the attributes mean and how they are supposed to be shaped 🙈 @chahank @tovogt, maybe you can help?

Reply:
In order to finalize the docstrings, we will probably need input from somebody who is familiar with this hazard type...

Reply:
True. I hope @mmyrte can help here. 🙏